code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -2846,3 +2854,72 @@
 	for(var/i = 1, i <= created_volume, i++)
 		new /obj/item/stack/material/diamond(location)
 	return
+
+//Temperature Additions
+/datum/chemical_reaction/water_to_ice
+	name = "Water to Ice"
+	id = "water_to_ice"
+	result = "ice"
+	required_reagents = list("water" = 1)
+	required_temperatures_max = list("water" = T0C - 5)
+	result_amount = 1
+
+/datum/chemical_reaction/ice_to_water
+	name = "Ice to Water"
+	id = "ice_to_water"
+	result = "water"
+	required_reagents = list("ice" = 1)
+	required_temperatures_min = list("ice" = T0C + 3)
+	result_amount = 0.2
+
+/datum/chemical_reaction/phoron_salt

```

Comment:
I am still not really sold on adding this. You said it's a precursor chem, but it's not a precursor for anything at the moment. Is it just to show off the heating mechanics?

Reply:
Phoron salt will be a precursor chem for chems to come. Right now it alone as a thing that science can mess around with is good enough.